templates/spaces/new.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<meta charset="utf-8" />
+    <head>
+        <title>MakersBnB</title>
+
+    </head>
+    <body>
+        <a class="link link_spaces" href="/spaces">Spaces</a> | <a class="link link_requests" href="/requests">Requests</a> | <a class="link link_homepage" href="/">Sign out</a> 
+        <h1>List your space</h1>
+        <form action="/spaces/new" method="post">
+            <p>
+            <label for="title">Title:</label><br>
+            <input type="text" name="title" id="title">
+            </p>
+
+            <p>
+            <label for="description">Description:</label><br>
+            <input type="text" name="description" id="description" style="width: 100mm;">
+            </p>
+
+            <p>
+            <label for="price">Price Per Night:</label><br>
+            <input type="number" name="price_per_night" id="price">
+            </p>
+
+            <p>
+            <label for="email">Email Address:</label><br>
+            <input type="text" name="email_address" id="email_address">
```

Comment:
We could make the input type for email_address an email, for automatic validation purposes

Reply:
Changed input type to email